bpython/autocomplete.py

```diff
@@ -544,10 +544,14 @@ def get_completer(completers, cursor_offset, line, **kwargs):
             double underscore methods like __len__ in method signatures
     """
 
-    for completer in completers:
-        matches = completer.matches(cursor_offset, line, **kwargs)
-        if matches is not None:
-            return sorted(matches), (completer if matches else None)
+    try:
+        for completer in completers:
+            matches = completer.matches(cursor_offset, line, **kwargs)
+            if matches is not None:
+                return sorted(matches), (completer if matches else None)
+    except:
```

Comment:
`except:` is certainly wrong here. This should not catch `KeyboardInterrupt` and friends. Catching the exception should also happen in Repl or the UI so that a message can be displayed.